pkg/repo/fs.go

```diff
@@ -135,6 +140,16 @@ func (fsr *FsRepo) Open() error {
 		}
 	}
 
+	// check if an installationID exists attempting to persist one if not found.
+	// never fail here as this isn't critical to node start up.
+	if installationID, err := fsr.ReadInstallationID(); err != nil {
+		log.Trace().Err(err).Msgf("failed to read installationID")
+	} else if installationID == "" {
+		if err := fsr.WriteInstallationID(ReadInstallationID()); err != nil {
+			log.Trace().Err(err).Msg("failed to write installationID")
+		}
+	}
+
```

Comment:
why are we persisting one if not found? As we discussed before, installationID should only be provided by the installation script and we won't generate one on our own when running bacalhau. That will be handled by the instanceID